pymilvus/orm/collection.py

```diff
@@ -598,6 +598,8 @@ def search(self, data, anns_field, param, limit, expr=None, partition_names=None
         :type  param: dict
         :param limit: The max number of returned record, also known as ``topk``.
         :type  limit: int
+        :param round_decimal: The specified number of decimal places of returned distance
+        :type  round_decimal: int
```

Comment:
Remember to change&check the example code.

Clearly this's outdated and cannot run